src/components/App/App.js

```diff
@@ -28,13 +32,18 @@ const App = () => {
     getWeather(location)
     .then(res => {
       setWeather(res.data.values);
+      getWeatherType();
       history.push(`/city/${location}`)
     })
     .catch(() => setError(true));
   };
 
+  const getWeatherType = () => {
+    setWeatherType(weatherCodes[weather.weatherCode])
```

Comment:
Appreciate the structure of this object - easy to reference the property you need via key